### DISCUSSION TITLE
Render css editor with angular

### DISCUSSION THREAD

**Discussion by everton-pessoa** [GrapesJS/grapesjs#3530] [Q&A]
Hi,
Could someone help me render css
customized on
editor with Angular?

I tried with canvas:

```
canvas: {
  styles: ['.test/test.component.css']
}

```
 but I get the following error:
 
`Refused to apply style from 'http://localhost:4200/.test/test.component.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.`
 
 thank's

**Comment by guigomes29** [comment]
Just to add to what he said, what we are basically trying is to add material components to the editor. The components are loading but without styling. We tried loading material stylesheet through angular.json and through the grapesjs editor.init function, using canvas { styles: [] }.
Anyone tried something similar with success?

**Reply by ronaldohoch** [reply]
Have you tried to load the full path of the material design css theme?

**Comment by artf** [comment]
Well that an issue with your environment, that has nothing to do with the editor, just find the proper way to load your CSS

**Comment by ronaldohoch** [comment]
Try to create a stackblitz, so we can help.
But i think you can create a component and add the script with a service doing lazyload of the script 🤔

**Comment by everton-pessoa** [comment]
Interestingly I can load an external css (CDN) but I can't load an internal one, as @artf  mentioned probably the problem is with my code, I must not be following the way expected by Angular.

Thanks for the help @artf  and @ronaldohoch . 

Thanks for the excellent editor, he makes me very happy.
